Makefile

```diff
@@ -2,28 +2,53 @@
 ## Install gcn2scimma
 ##
 INSTALL_DIR   := /usr/local/gcn2hop
-PYTHON_FILES  := gcn2hop Utils.py 
+WORKING_DIR	  := ./gcn2hop
+PYTHON_FILES  := gcn2hop,Utils.py
 INSTALL_FILES := $(patsubst %,$(INSTALL_DIR)/%,$(PYTHON_FILES))
```

Comment:
This shouldn't be necessary either after moving to a module.

Reply:
I just aimed to keed it this way as well in case we do not need to make changes in the container